### PR TITLE
[#97] bug fix: can't get response when using charset error.

### LINKED ISSUE #97: [BUG] can't get response when using content-type:application/json; utf-8
**Describe the bug**
I followed tutorial and found a bug
when I request server using 
curl -X GET -H "Content-Type: application/json; charset=utf-8" -d '{"name" : "sale"}' http://127.0.0.1:8181/my/service,
I got error that content-type is null
```
2020-08-05 15:15:15 [INFO ] [DssRestMasterActor.java]handlingDssRestMasterActorCommandRequest(55) : DssRestMasterActorCommandRequest: DssRestMasterActorCommandRequest{channelId: '982cbcfffea32609-00000370-00000001-0a7937d2f1227434-bb86d31d', sender: 'rest-handler-1', methodType: 'GET', contentType: 'null', path: '/my/service', content: '{"name" : "sale"}'}
2020-08-05 15:15:15 [INFO ] [DssRestServiceActor.java]onHandlingDssRestServiceActorCommandRequest(42) : onHandlingDssRestServiceActorCommandRequest: DssRestMasterActorCommandRequest{channelId: '982cbcfffea32609-00000370-00000001-0a7937d2f1227434-bb86d31d', sender: 'rest-handler-1', methodType: 'GET', contentType: 'null', path: '/my/service', content: '{"name" : "sale"}'}
2020-08-05 15:15:15 [WARN ] [DssRestServiceActor.java]onHandlingDssRestServiceActorCommandRequest(44) : Request content-type is null: DssRestMasterActorCommandRequest{channelId: '982cbcfffea32609-00000370-00000001-0a7937d2f1227434-bb86d31d', sender: 'rest-handler-1', methodType: 'GET', contentType: 'null', path: '/my/service', content: '{"name" : "sale"}'}
2020-08-05 15:15:15 [ERROR] [DssRestServiceActor.java]onHandlingDssRestServiceActorCommandRequest(52) : Cannot handling request for unexpected content-type. Expected: APPLICATION_JSON, but: null
2020-08-05 15:15:15 [INFO ] [DssRestHandler.java]handlingDssRestChannelHandlerCommandResponse(212) : DssRestChannelHandlerCommandResponse: DssRestChannelHandlerCommandResponse{channelId: '982cbcfffea32609-00000370-00000001-0a7937d2f1227434-bb86d31d', status: '400', response: 'null'}
```

### PR DESCRIPTION
**Pull request by RulLu16**
Resolves #97 

fix bug when using charset in content type like 
curl -X GET -H "Content-Type: application/json; charset=utf-8" -d '{"name" : "myname"}' http://127.0.0.1:8181/my/service.

I just split content-type text into application/json and charset=utf-8.
And for setting charset, add charset member in DssRestRequest.

### CODE REVIEW

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
     }
 
     public static DssRestContentType fromText(String text) {
-        return textMap.get(text);
+    	return textMap.get(text.split(";")[0]);
```
Comment (ztkmkoo) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java:37]: Hi, if text can be null?

Comment (RulLu16) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java:37]: Oh, I didn't consider that text can be null. I'll fix this. Thank you!

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java [hunk 2]
```diff
@@ -33,8 +33,12 @@
...
     }
 
-    public static DssRestContentType fromText(String text) {
-        return textMap.get(text);
+    public static DssRestContentType fromText(String text) {    	
+    	try{
+    		return textMap.get(text.split(";")[0]);
+    	}catch(NullPointerException e) {
+    		return textMap.get(text);
+    	}
```
Comment (RulLu16) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java:41]: thanks to @Alencion's suggestion,
I think 

`
if (Objects.nonNull(text)){
        return textMap.get(text.split(";")[0]);
        }
        return null;
`
is easier to understand the code.

What do you think? @ztkmkoo

Comment (ztkmkoo) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java:41]: I think check null looks better than catch the exception.

Comment (RulLu16) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/network/rest/enumeration/DssRestContentType.java:41]: I agree with you. I'll modify it.

**Pull request merged**